Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -122,127 +122,99 @@ public final class SecureEnclaveValet: NSObject {
     /// - Parameters:
     ///   - object: A Data value to be inserted into the keychain.
     ///   - key: A key that can be used to retrieve the `object` from the keychain.
-    /// - Returns: `false` if the keychain is not accessible.
+    /// - Throws: An error of type `KeychainError`.
     @objc
-    @discardableResult
-    public func setObject(_ object: Data, forKey key: String) -> Bool {
-        execute(in: lock) {
-            guard let keychainQuery = keychainQuery else {
-                return false
-            }
-            return SecureEnclave.setObject(object, forKey: key, options: keychainQuery)
+    public func setObject(_ object: Data, forKey key: String) throws {
+        try execute(in: lock) {
+            try SecureEnclave.setObject(object, forKey: key, options: try keychainQuery())
         }
     }
 
     /// - Parameters:
     ///   - key: A key used to retrieve the desired object from the keychain.
-    ///   - userPrompt: The prompt displayed to the user in Apple's Face ID, Touch ID, or passcode entry UI.
-    /// - Returns: The data currently stored in the keychain for the provided key. Returns `.itemNotFound` if no object exists in the keychain for the specified key, or if the keychain is inaccessible. Returns `.userCancelled` if the user cancels the user-presence prompt.
-    public func object(forKey key: String, withPrompt userPrompt: String) -> SecureEnclave.Result<Data> {
-        execute(in: lock) {
-            guard let keychainQuery = keychainQuery else {
-                return .itemNotFound
-            }
-            return SecureEnclave.object(forKey: key, withPrompt: userPrompt, options: keychainQuery)
+    ///   -  userPrompt: The prompt displayed to the user in Apple's Face ID, Touch ID, or passcode entry UI.
+    /// - Returns: The data currently stored in the keychain for the provided key.
+    /// - Throws: An error of type `KeychainError`.
+    @objc
+    public func object(forKey key: String, withPrompt userPrompt: String) throws -> Data {
+        try execute(in: lock) {
+            try SecureEnclave.object(forKey: key, withPrompt: userPrompt, options: try keychainQuery())
         }
     }
 
     /// - Parameter key: The key to look up in the keychain.
-    /// - Returns: `true` if a value has been set for the given key, `false` otherwise. Will return `false` if the keychain is not accessible.
-    /// - Note: Will never prompt the user for Face ID, Touch ID, or password.
-    @objc
-    public func containsObject(forKey key: String) -> Bool {
-        execute(in: lock) {
-            guard let keychainQuery = keychainQuery else {
-                return false
-            }
-            return SecureEnclave.containsObject(forKey: key, options: keychainQuery)
+    /// - Returns: `true` if a value has been set for the given key, `false` otherwise.
+    /// - Note: Will never prompt the user for Face ID, Touch ID, or password. Method will throw a `KeychainError`.
```

Comment:
Let's use `Throws` instead of `Note` for the KeychainError comment.

Reply:
Ah! Missed one! Nice catch 🙂.

Resolved in e7023c9